<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyProgressWindow</class>
 <widget class="QDialog" name="CopyProgressWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="copyStatusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>备份中</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="copyProgressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>480</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="TitleBar" name="titleBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="copyLogTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>149</y>
     <width>480</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="copyCountLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>390</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="copyCurrentPathLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>390</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>current</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyStopPushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QGroupBox</extends>
   <header location="global">src/utils/widget/titlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
